set.mm

```diff
@@ -496112,125 +496221,216 @@ such that every prime ideal contains a prime element (this is a
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
-  $c AlgNb minPoly $.
-
-  $( Extend class notation with the algebraic number builder function. $)
-  calgnb $a class AlgNb $.
+  $c IntgRing $. $( Integral subring of a ring $)
+  $c minPoly $. $( Minimal polynomial $)
 
-  $( Extend class notation with the minimal polynomial builder function. $)
-  cminply $a class minPoly $.
+  $( Integral subring of a ring. $)
+  cirng $a class IntgRing $.
 
   ${
-    $d e f p $.
-    $( Define the algebraic number builder function.  This definition is
-       similar to ~ df-aa .  (Contributed by Thierry Arnoux, 19-Jan-2025.) $)
-    df-algnb $a |- AlgNb = ( e e. _V , f e. _V |-> U_ p e. ( dom ( e evalSub1 f
-       ) \ { ( 0g ` ( Poly1 ` e ) ) } ) ( `' ( ( e evalSub1 f ) ` p ) " { ( 0g
-       ` e ) } ) ) $.
+    $d f r s $.
+    $( Define the subring of elements of ` r ` integral over ` s ` in a ring.
+       (Contributed by Mario Carneiro, 2-Dec-2014.)  (Revised by Thierry
+       Arnoux, 28-Jan-2025.) $)
+    df-irng $a |- IntgRing = ( r e. _V , s e. _V |->
```

Comment:
"Define the subring of elements of ` r ` integral over ` s ` in a ring." -> Define the subring of elements of a ring ` r ` integral over ` s `."

Reply:
This was Mario's original description verbatim, but I changed nevertheless.